lib/trace-of-tab.js

```diff
@@ -20,7 +20,7 @@ const log = require('debug')(`tracium`);
 const TracingProcessor = require('./tracing-processor.js');
 const LHError = require('./lh-error.js');
 
-const ACCEPTABLE_NAVIGATION_URL_REGEX = /^(chrome|https?):/;
+const ACCEPTABLE_NAVIGATION_URL_REGEX = /^(chrome|https|file?):/;
```

Comment:
This should be `/^(chrome|https?|file):/`

the `?` is for the optional `s` on `https` :)

Reply:
Yep)
You right. My mistake.

Reply:
I updated code.